src/main.rs

```diff
@@ -106,6 +106,38 @@ struct Args {
 }
 
 fn main() -> anyhow::Result<()> {
+    // Use `console` and/or `tracy` features to enable both `tracing-subscriber` and the layer(s)
+    #[cfg(feature = "tracing-subscriber")]
```

Comment:
Maybe we can extract this into a separate function to make it a _bit nicer_.

Reply:
+1